dev/CommonStyles/TextBox_themeresources.xaml

```diff
@@ -25,11 +25,12 @@
             <SolidColorBrush x:Key="TextBoxDisabledBorderThemeBrush" Color="#66FFFFFF" />
             <SolidColorBrush x:Key="TextBoxDisabledForegroundThemeBrush" Color="#FF666666" />
             <SolidColorBrush x:Key="TextBoxForegroundThemeBrush" Color="#FF000000" />
-            <StaticResource x:Key="TextControlBackgroundFocused" ResourceKey="SystemAltHighColor" />
+            <StaticResource x:Key="TextControlBackgroundFocused" ResourceKey="SystemControlBackgroundAltHighBrush" />
             <StaticResource x:Key="TextControlBorderBrush" ResourceKey="SystemControlForegroundBaseMediumBrush" />
             <StaticResource x:Key="TextControlBorderBrushPointerOver" ResourceKey="SystemControlHighlightBaseMediumHighBrush" />
-            <StaticResource x:Key="TextControlButtonForeground" ResourceKey="SystemBaseHighColor" />
-            <StaticResource x:Key="TextControlForegroundFocused" ResourceKey="SystemBaseHighColor" />
+            <StaticResource x:Key="TextControlButtonForeground" ResourceKey="SystemControlPageTextBaseHighBrush" />
+            <StaticResource x:Key="TextControlForegroundFocused" ResourceKey="SystemControlPageTextBaseHighBrush" />
```

Comment:
`SystemControlPageTextBaseHighBrush` and `SystemControlForegroundBaseHighBrush` do have the same color by default, but `TextControlForeground` references `SystemControlForegroundBaseHighBrush`, so it may be a good idea to be consistent.

Reply:
Thanks switched to that brush now